LykkeApi2/Services/KycCheckService.cs

```diff
@@ -0,0 +1,47 @@
+using System.Threading.Tasks;
+using Core.Services;
+using Lykke.Service.Kyc.Abstractions.Domain.Verification;
+using Lykke.Service.Kyc.Abstractions.Services;
+using Lykke.Service.Tier.Client;
+
+namespace LykkeApi2.Services
+{
+    public class KycCheckService : IKycCheckService
+    {
+        private readonly IKycStatusService _kycStatusService;
+        private readonly ITierClient _tierClient;
+
+        public KycCheckService(
+            IKycStatusService kycStatusService,
+            ITierClient tierClient
+        )
+        {
+            _kycStatusService = kycStatusService;
+            _tierClient = tierClient;
+        }
+        public async Task<bool> IsKycNeededAsync(string clientId)
```

Comment:
I have seen similar method in previous PRs related to this feature.

Probably it will be nice to move to KYC service + reuse.